Source/ASCellNode.h

```diff
@@ -87,7 +88,7 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
  * @return The supplementary element kind, or @c nil if this node does not represent a supplementary element.
  */
 //TODO change this to be a generic "kind" or "elementKind" that exposes `nil` for row kind
-@property (nonatomic, copy, readonly, nullable) NSString *supplementaryElementKind;
+@property (copy, readonly, nullable) NSString *supplementaryElementKind;
```

Comment:
I know it is implied, but stylistically, I think it would be a good idea to require atomic to be stated explicitly for properties that intend it. Although it might become the more common thing, currently it's uncommon enough that most cases where it is omitted could be confused for mistakes. Any thoughts?